Makefile

```diff
@@ -2,10 +2,13 @@ all: build
 
 TAG = v0.18.0
 PREFIX = gcr.io/google_containers
-FLAGS = 
+FLAGS =
 
 deps:
 	go get github.com/tools/godep
+	go get github.com/progrium/go-extpoints
```

Comment:
This should not be here since we use the vendored go-extpoints version as seen in https://github.com/kubernetes/heapster/blob/8e4a8ba63e75fe234bedb4171b35553d6ea18981/Godeps/Godeps.json#L6